content/docs/guides/contribute/contribute-code/frontend-conventions.fr.md

```diff
@@ -213,58 +220,67 @@ Dans la terminologie de RTK query, Lorsqu’un composant react appelle un endpoi
 RTK compte le nombre de référence à la même paire (endpoint,{paramètres}). Lorsque deux composants souscrivent à la même donnée. Ils partagent la même clé dans le cache.
 
 ```ts
-import { osrdEditoastApi } from './api.ts'  
-  
-function Component1() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(1)
-  
-  return <div>...</div>  
-}  
-  
-function Component2() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(2)
-  
-  return <div>...</div>  
-}  
-  
-function Component3() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(3)  
-  
-  return <div>...</div>  
-}  
-  
-function Component4() {  
-  // component subscribes to the *same* data as ComponentThree,  
-  // as it has the same query parameters  
-  const { data } = osrdEditoastApi.useGetXQuery(3)  
-  
-  return <div>...</div>  
+import { osrdEditoastApi } from "./api.ts";
+
+function Component1() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(1);
+
+  return <div>...</div>;
+}
+
+function Component2() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(2);
+
+  return <div>...</div>;
+}
+
+function Component3() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(3);
+
+  return <div>...</div>;
+}
+
+function Component4() {
+  // component subscribes to the *same* data as ComponentThree,
+  // as it has the same query parameters
+  const { data } = osrdEditoastApi.useGetXQuery(3);
+
+  return <div>...</div>;
 }
 ```
 
-Ici `Component3` et `Component4` ne vont générer qu’un seul appel vers le back. Ils souscrivent à la même donnée (même endpoint et même paramètre `3`). Ils vont partager la même clé dans le cache. 
+Ici `Component3` et `Component4` ne vont générer qu’un seul appel vers le back. Ils souscrivent à la même donnée (même endpoint et même paramètre `3`). Ils vont partager la même clé dans le cache.
 
 Au total ici il y aura trois appels vers le back, avec les paramètres `1`, `2`, `3`.
 
 Tant qu’il existe au moins un composant react monté, qui appelle le hook `osrdEditoastApi.endpoints.getProjectsByProjectId.useQuery` par exemple, la donnée sera conservée dans le cache.
 
 Dès que le dernier composant est démonté, la donnée est supprimée du cache au bout de 60 secondes (valeur par défaut).
 
+### Traduction de l'application
+
+La traduction de l'application est effectuée sur [Transiflex]. La langue par défaut est le français.
+A noter que si l'on doit corriger une traduction, il est recommandé de passer par Transiflex. En revanche, si on ajoute une nouvelle clé de traduction, celle-ci peut être ajoutée dans le code directement, dans toutes les langues disponibles.
```

Comment:
```suggestion
La traduction de l'application est effectuée sur [Transifex]. La langue par défaut est le français.
A noter que si l'on doit corriger une traduction, il est recommandé de passer par Transifex. En revanche, si on ajoute une nouvelle clé de traduction, celle-ci peut être ajoutée dans le code directement, dans toutes les langues disponibles.
```

Reply:
Done, I also added the link as I had forgotten it